### PR TITLE
Add Preupdate and Postupdate api decorator 

### PR DESCRIPTION
**Pull request by rga-odoo**
task: https://www.odoo.com/web#model=project.task&id=30104&active_id=30104&view_type=form&menu_id=
pad: https://pad.odoo.com/p/r.e9ffe510971f48e51bec664c15ebafc7

### CODE REVIEW

File: odoo/api.py [hunk 1]
```diff
@@ -42,7 +42,7 @@
     'cr_uid_ids', 'cr_uid_ids_context',
     'cr_uid_records', 'cr_uid_records_context',
     'constrains', 'depends', 'onchange', 'returns',
-    'call_kw',
+    'call_kw', 'prewrite', 'postupdate'
```
Comment (langbamit) [inline odoo/api.py]: preupdate

**Comment by Yenthe666** [comment]
Hi,

Can we also expect documentation for this? It is not documented anywhere now I believe.

**Comment by naglis** [comment]
Does this have a chance of making it into v13? Would be great :slightly_smiling_face:

**Comment by pedrobaeza** [comment]
Is this going to be continued?

**Comment by C3POdoo** [comment]
Dear @rga-odoo,

Thank you for your report but we are closing it due to inactivity.
We apology if we could not look at your request in time.
If your report still makes sense, don't hesitate to reopen a new one. We will try to check it as soon as possible.

_This is an automated message._

**Pull request closed without merging**